### ISSUE TITLE
Duplication Bug: Spectre Anchor + Soulbound(CoFH)

### ISSUE THREAD

**Issue by Dalois** [lumien231/Random-Things#433]
When an item is both enchanted with soulbound and anchored, and the player dies, they will respawn with 2 of the item in their inventory.

Minecraft version = 1.12.2
CoFH Core = CoFHCore-1.12.2-4.6.3.27-universal.jar
Random Things version = RandomThings-MC1.12.2-4.2.7.3.jar
Forge version = 14.23.5.2838

**Comment by Sereath** [comment]
Yeah, can confirm that CoFH's soulbound enchant + Spectre Anchor dupes items.
Forge 2860
Random Things 4.2.7.4
